src/Modules/Admin/Commands.Admin/help/Add-PowerBIEncryptionKey.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version:
+schema: 2.0.0
+---
+
+# Add-PowerBIEncryptionKey
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
Fill in

Reply:
Temporarily filled in with the docs from spec. Will update this before going public